package.json

```diff
@@ -43,7 +43,7 @@
     "@apollo/client": "^3.5.6",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
-    "@sentry/browser": "^6.16.1",
+    "@sentry/browser": "^7.0.0",
```

Comment:
Note to self: include `BREAKING CHANGES: bump @sentry/browser depency to version 7` in the squashed commit message.